src/main/java/uk/gov/hmcts/sptribs/citizen/event/CicDssUpdateCaseEvent.java

```diff
@@ -128,33 +128,40 @@ private CaseData addDocumentsToCaseData(final CaseData caseData, final DssCaseDa
                 if (!documentList.contains(caseworkerCICDocument)) {
                     documentList.add(caseworkerCICDocument);
                 }
-            }
-        }
+                if (isNotBlank(dssCaseData.getAdditionalInformation()) || isNotBlank(documentComment)) {
```

Comment:
This file does not need updating as it already includes doc relevance comments per document. You need to update CicUpdateCaseEvent which is used by Dss Submit journey